lib/commands/context.js

```diff
@@ -98,7 +98,7 @@ commands.setContext = async function setContext (name, callback, skipReadyCheck)
   }
 
   log.debug(`Attempting to set context to '${name}' from '${this.curContext ? this.curContext : NATIVE_WIN}'`);
-  if (alreadyInContext(name, this.curContext) || alreadyInContext(name.replace(WEBVIEW_BASE, ''), this.curContext)) {
+  if (alreadyInContext(name, this.curContext) || (name !== null && alreadyInContext(name.replace(WEBVIEW_BASE, ''), this.curContext))) {
```

Comment:
you could use lodash's '_.replace', which properly handles null arguments

Reply:
oh, great... lodash has many helpful methods 🎉

Reply:
yep, I would also add a comment there about `name` could be null, because it is not obvious in that context